content/docs/concepts/traffic-management/destination-rules/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Destination Rules
+description: Describes the architecture and behavior of destination rules and how Istio enforces them.
+weight: 4
+keywords: [destination, request, route, policy, subset, destination rule, TLS, evaluation, failure handling, fault injection, traffic routing, gateway]
+---
+
+A [destination rule](/docs/reference/config/istio.networking.v1alpha3/#DestinationRule)
+configures the set of policies **Pilot** applies to a request **after** the
+[virtual service](../virtual-services) routing.
+
+Service owners typically author destination rules to configure, among other
+things:
+
+- [Circuit breakers](../failures/#circuit)
+- TLS settings
+- Outlier detection
+- [Load balancer](../load-balancing) settings
+- Definition of [subsets](../traffic-routing) of destination hosts in the mesh
+
+Use wild card prefixes in a destination rule to specify a single rule for
+multiple services. Istio uses the defined subsets of the destination hosts in
+the virtual services  `route:` specification to send traffic to specific
+versions of the service. This naming of versions allows you to:
+
+-  Cleanly refer to a specific service version across different virtual
+   services.
+-  Simplify the stats that the Istio proxies emit.
+-  Encode subsets in SNI headers.
+
+The following example destination rule configures three different subsets with
+different load balancing policies for the `vtl-svc` virtual service:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: my-destination-rule
+spec:
+  host: my-vtl-svc
+  trafficPolicy:
+    loadBalancer:
+      simple: RANDOM
+  subsets:
+  - name: v1
+    labels:
+      version: v1
+  - name: v2
+    labels:
+      version: v2
+    trafficPolicy:
+      loadBalancer:
+        simple: ROUND_ROBIN
+  - name: v3
+    labels:
+      version: v3
+{{< /text >}}
+
+Notice the multiple `trafficPolicy` entries. You can specify multiple policies
+in a single `DestinationRule` configuration file. In this example, the default
+policy is defined above the `subsets:` key and the `v2` specific policy withing
+the appropriate subset.
+
+The following diagram shows how the different configurations in the
+`my-destination-rule` destination rule affect the traffic to and from the
+`my-vtl-svc` virtual service:
+
+{{< image width="50%"
+    link="./destination-rule.svg"
+    caption="Configurable route examples defined in the destination rule"
+    >}}
+
+Visit our [destination rules reference documentation](/docs/reference/config/istio.networking.v1alpha3/#DestinationRule)
+to review all the enabled keys and values.
+
+## Destination rules routing evaluation
+
+Similar to route rules in virtual services, **Pilot** associates policies
+defined in a destination rule with a particular `host:` service. If the
+policies are subset specific, the activation depends on the route rule
+evaluation results.
+
+The rule evaluation process is as follows:
```

Comment:
All 4 of the following steps are wrong. I also don't think that describing the rule evaluation algorithm in the Destination rule section of the docs makes sense.